eval/fusion.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+'''
```

Comment:
The boilerplate here got mangled somewhere (other script too)? Use this one: https://github.com/castorini/anserini/blob/master/src/main/python/run_solr_regression.py#L2

Reply:
Copy/paste messed up for some reason. Addressed